soc/xtensa/esp32/Kconfig.soc

```diff
@@ -246,4 +250,25 @@ config ESP32_UNIVERSAL_MAC_ADDRESSES
 	default 2 if ESP32_UNIVERSAL_MAC_ADDRESSES_TWO
 	default 4 if ESP32_UNIVERSAL_MAC_ADDRESSES_FOUR
 
+config ESP_BOOTLOADER_SIZE
+	hex
```

Comment:
What is this setting ?
Where is it used and why.

Who should set this value.
Why is it promptless.

Same goes for the settings below.

Reply:
Not used, closing this thread.